Automattic/pocket-casts-android
# Enable swipe up in now playing landscape

## Description

Fixes: Swiping up to get to "Up Next" doesn't work when unfolded (p1690329518000709-slack-C028JAG44VD)
Test Device : Pixel Fold


## Testing Instructions
1. Open the app on Pixel Fold emulator - landscape mode
3. Play an episode
4. Go to Now Playing
5. Swipe up
6. Notice that Up Next is shown
7. Repeat steps 3-6 after fold/ unfold phone
8. Notice that Up Next is still shown

**  The issue is not limited to foldable devices. Swiping up to get to "Up Next" on Now Playing never worked in landscape mode, and I just enabled it.

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/53b9d9dc-2049-4b81-a9ce-e81b34a5c34c

I also tested that the screen is not shown for Automotive or Android Auto so it does not affect those platforms.

Automotive| Android Auto
----| ----
![Screenshot_20230726_112836](https://github.com/Automattic/pocket-casts-android/assets/1405144/3a6ed7c0-78e8-499a-b480-220eb98a545f)| <img  alt="Screenshot 2023-07-26 at 11 36 25 AM" src="https://github.com/Automattic/pocket-casts-android/assets/1405144/c5f62efa-d661-462b-bef9-a6aae55e275e">



## Checklist
 
#### I have tested any UI changes...
